### DISCUSSION TITLE
Textinput input suffix/prefix

### DISCUSSION THREAD

**Discussion by Jacobtims** [filamentphp/filament#10522] [Ideas]
Would be nice to have an option to add input fields as suffix or prefix.
`suffixInput()` or `prefixInput()`.

The only way to do it right now is using a grid, but alignment is not great.

Check example below, see checkbox unlimited.
![image](https://github.com/filamentphp/filament/assets/75219092/82a64026-4bc4-4ccc-a3f8-b315a8a92fb2)

**Comment by kayvanaarssen** [comment]
Would be an awesome feature!
